test/integration/swaggers/appconfiguration.json

```diff
@@ -0,0 +1,1347 @@
+{
```

Comment:
Would you mind either

1) Add a simple appconfig.spec.ts test that instantiates this client or
2) Add this to the smoke tests instead of integration

This is so that we also get build validation. We currently don't build the integration test unless they are being actually tested, while smoke-test generates and the builds the generated project

Reply:
Done